src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckSwitchExpressionsFour.java

```diff
@@ -0,0 +1,38 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java14
+package com.puppycrawl.tools.checkstyle.checks.coding.missingswitchdefault;
+
+class InputMissingSwitchDefaultCheckSwitchExpressionsFour {
+    public void switchWithNullTest(SwitchInput i) {
+        switch (i) {
+            case FIRST: i++;
+            case SECOND:  i++;
+            case null: i++;
+            case null: i++;
+        }
```

Comment:
Please take the time to learn syntax and check that your inputs are compilable; duplicate case labels and postfix operators are not valid here.

Reply:
Sorry for this changed @nrmancuso